packages/connect/src/device/DeviceList.ts

```diff
@@ -123,7 +123,7 @@ export class DeviceList extends TypedEmitter<DeviceListEvents> {
                         );
                 }
             } else if (transportType instanceof Transport) {
-                this.transports.unshift(transportType);
+                this.transports.push(transportType);
```

Comment:
makes sense in the end. lets keep transports to be used in the order provided by user. 
this might affect @szymonlesisz with bluetooth.